test/test_gpu2.cpp

```diff
@@ -8679,7 +8632,7 @@ TEST_F(NVFuserTest, FusionFloatPow_CUDA) {
       &fusion,
       outputs,
       aten_inputs,
-      {p4, p2, p2, p3, p3, t6},
+      // {p4, p2, p2, p3, p3, t6},
```

Comment:
Remove dead code

Reply:
Restored this, this should not be commented.